playbooks/tasks/portal-renter-wait-until-idle.yml

```diff
@@ -0,0 +1,69 @@
+# Wait 5 minutes for any small uploads and downloads to finish. After 5 minutes,
+# it is likely that it is a large upload in which case it is safe to take the
+# server down as the large upload will continue on another server. Or it is a
+# large download which can also be resumed on another server.
+
+# Check if sia container is running.
+- name: Check sia container is running
+  community.docker.docker_container_info:
+    name: sia
+  register: sia_docker_container_result
+
+# Get the ip address for the sia container
+- name: Get sia service IP inside docker compose network
+  command: docker inspect -f "{% raw %}{{ range.NetworkSettings.Networks }}{{ .IPAddress }}{{ end }}{% endraw %}" sia
+  register: sia_ip_result
```

Comment:
I think this will fail portal setup playbook on a fresh server.
I would add `ignore_errors: True`.